pkgs/tools/networking/inetutils/default.nix

```diff
@@ -48,6 +48,6 @@ stdenv.mkDerivation rec {
     license = stdenv.lib.licenses.gpl3Plus;
 
     maintainers = [ ];
-    platforms = stdenv.lib.platforms.gnu;
+    platforms = stdenv.lib.platforms.gnu ++ stdenv.lib.platforms.linux;
```

Comment:
This one should be stdenv.lib.platforms.unix (it works on Darwin)

Reply:
Thanks for reviewing! Okay, I'll address this shortly.  Hooray for working on more platforms!  :+1: